openassessment/xblock/schema.py

```diff
@@ -63,6 +63,12 @@ def datetime_validator(value):
     u'student-training'
 ]
 
+VALID_UPLOAD_FILE_TYPE = [
+    u'none',
+    u'image',
+    u'file',
```

Comment:
Why we are calling this `file`. Does this shows the pdf & image option if so, then we should name this `pdf-and-image` or something more suitable, `file` option is too generic for this.